pom.xml

```diff
@@ -1416,6 +1416,14 @@
                     <groupId>pl.project13.maven</groupId>
                     <artifactId>git-commit-id-plugin</artifactId>
                     <version>4.0.0</version>
+                    <configuration>
+                        <!-- Include only required properties to speed up build (https://github.com/git-commit-id/git-commit-id-maven-plugin/issues/462) -->
```

Comment:
Should we contribute that airbase?

Reply:
https://github.com/prestosql/presto/pull/5817#pullrequestreview-523429608